setup.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from setuptools import setup, find_packages
+from os import path
+
+
+UTILS_CV = "utils_cv"  # Utility folder name
+# UTILS_CV_PATH = path.join(path.abspath(path.dirname(__file__)), UTILS_CV)
+# README = path.join(UTILS_CV_PATH, "README.md")
+README = path.join(UTILS_CV, "README.md")
+exec(open(path.join(UTILS_CV, "__init__.py")).read())
+
+with open(README, encoding="utf-8") as f:
+    long_description = f.read()
+
+setup(
+    name="utils_cv",
+    version=__version__,
+    description="Computer Vision Utilities",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/microsoft/ComputerVision",
+    project_urls={
+        "Bug Tracker": "https://github.com/microsoft/ComputerVision/issues",
+        "Source Code": "https://github.com/microsoft/ComputerVision",
+        "Documentation": "https://github.com/microsoft/ComputerVision",
+    },
+    author="CVDev Team at Microsoft",
+    classifiers=[
+        "Development Status :: 2-Pre-Alpha",
+        "Intended Audience :: Developers",
+        "Intended Audience :: Science/Research",
+        "License :: OSI Approved :: MIT License",
+        "Programming Language :: Python :: 3",
+        "Topic :: Scientific/Engineering",
+        "Topic :: Software Development :: Libraries :: Python Modules",
+    ],
+    include_package_data=True,
+    # Not fully test the dependency coverage, please make sure to sync
+    # with environment.yml
+    install_requires=[
+        "azureml-sdk[notebooks,contrib]>=1.0.30",  # requires ipykernel, papermill, jupyter-core, jupyter-client
```

Comment:
do we actually want to put the azureml sdk into our lib?

Reply:
`utils_cv/common/deployment.py` and `utils_cv/common/azureml.py` depends on `azureml-sdk`.

Reply:
oh right. arguably we should not support these. but maybe we can deal with that in the future... 

But i think it doesn't make sense that we're providing these additions on top of azureml's sdk. If we are modifying the user experience of azureml because it is not good enough as is, that code should live in the azureml sdk... otherwise, we should stick to the way azureml docs recommends.

Reply:
Yeah, we can deal with that in the future.